modules/images-configuration-allowed.adoc

```diff
@@ -188,9 +191,8 @@ The following policy indicates that only images from the example.com, quay.io, a
    }
 }
 ----
-====
 endif::openshift-rosa,openshift-dedicated[]
-
++
 [NOTE]
```

Comment:
In the OSD preview, the previous important note and this one are indented differently, which looks a bit odd. 

<img width="971" height="516" alt="Image" src="https://github.com/user-attachments/assets/65accb42-e314-4071-8467-3ad4e02f5f8f" />

Reply:
hmm let's try an open block on that one